src/main/java/org/openlmis/stockmanagement/service/StockCardService.java

```diff
@@ -104,6 +105,10 @@ public StockCard getStockCardById(Long facilityId, Long stockCardId) {
     return repository.getStockCards(facilityId);
   }
 
+  public Product getProductByStockCardId(Long stockCardId) {
```

Comment:
I've seen (and therefore repeated) this pattern in OpenLMIS, wherein a method added to a service does nothing other than wrap a call with the same method signature within a repository. The added code doesn't seem to add value, though, and thus doesn't seem ideal.